src/extensions/surveys/survey-event-receiver.ts

```diff
@@ -1,11 +1,11 @@
-import { Survey } from '../posthog-surveys-types'
-import { PostHogPersistence } from '../posthog-persistence'
-import { SURVEYS_ACTIVATED } from '../constants'
-import { CaptureResult } from '../types'
+import { Survey } from '../../posthog-surveys-types'
+import { PostHogPersistence } from '../../posthog-persistence'
+import { SURVEYS_ACTIVATED } from '../../constants'
+import { CaptureResult, IReceiveSurveyEvents } from '../../types'
 
-export class SurveyEventReceiver {
-    private readonly eventRegistry: Map<string, string[]>
-    private readonly persistence?: PostHogPersistence
+export class SurveyEventReceiver implements IReceiveSurveyEvents {
+    readonly eventRegistry: Map<string, string[]>
```

Comment:
ah, pycharm auto made these things not private for me when i extracted the interface here
that wasn't intentional